logentries/templates/default/logentries.conf.erb

```diff
@@ -1,10 +1,10 @@
 # Send entries to logentries via TCP.
-$DefaultNetstreamDriverCAFile /etc/ssl/certs/logentries.all.crt
+$DefaultNetstreamDriverCAFile /etc/ssl/certs/ca-bundle.crt
 
 $ActionSendStreamDriver gtls
 $ActionSendStreamDriverMode 1
 $ActionSendStreamDriverAuthMode x509/name
 $ActionSendStreamDriverPermittedPeer *.logentries.com
 
-$template LogentriesFormat,"<%= @token %> %HOSTNAME% %syslogtag%%msg%\n"
-*.* @@api.logentries.com:20000;LogentriesFormat
+$template LogentriesTemplate,"<%= @token %> %HOSTNAME% %syslogtag%%msg%\n"
+*.* @@data.logentries.com:443;LogentriesTemplate
```

Comment:
Logentries support said api is a deprecated endpoint and in https://docs.logentries.com/docs/rsyslog#section-secure-logging the example config uses data